### ISSUE TITLE
Mixed Studies 

### ISSUE THREAD

**Issue by DSWallach** [clemente-lab/mmeds-meta#231]
Figure out how to accept studies that include both human and animal samples

**Comment by adamcantor22** [comment]
@cleme Still desireable?

**Comment by cleme** [comment]
Yes. The scenario is actually (somewhat) common: germ-free mouse model is colonized with human microbiota, and samples are later collected from the mice. You would have human samples (used to colonize the mice) and mice samples after colonization.
